Sources/SwiftSyntaxMacrosTestSupport/Assertions.swift

```diff
@@ -254,16 +254,24 @@ func assertDiagnostic(
 ///   - macros: The macros that should be expanded, provided as a dictionary
 ///     mapping macro names (e.g., `"stringify"`) to implementation types
 ///     (e.g., `StringifyMacro.self`).
+///   - applyFixIts: Applies only the fix-its with these messages.
+///     Ohterwise all Fix-Its from diagnostics will be applied.
+///   - fixedSource: Asserts that the source after applying fix-its matches
```

Comment:
```suggestion
///   - fixedSource: The source code to expect after applying the Fix-Its matching `applyFixIts`.
```

Reply:
After added this comment, I tought that `Ohterwise all Fix-Its from diagnostics will be applied.` didn't make sense. 
So tried to make it a bit more explicit. 

Can you prof read again?